Softeq.XToolkit.WhiteLabel.Forms/Navigation/FormsNavigationService.cs

```diff
@@ -77,7 +74,7 @@ public void GoBack()
             Navigation.PopAsync().FireAndForget(_logger);
         }
 
-        private async Task NavigateTask(
+        private async Task NavigateTaskAsync(
```

Comment:
Seems like "Task" part in the name is really unnecessary here. "NavigateAsync" should be enough